bundles/org.openhab.binding.zwavejs/README.md

```diff
@@ -0,0 +1,91 @@
+
+## Supported Things
+
+This binding supports the following types of things:
+
+- `gateway`: Represents the Z-Wave JS Webservice bridge. This is required to communicate with the Z-Wave network.
+- `node`: Represents a Z-Wave device (node) in the network. Each node can have multiple channels corresponding to its capabilities.
+
+**Note**: This binding does not maintain a Z-Wave device database and relies on the external Z-Wave JS project for device compatibility and functionality.
+
+## Discovery
+
+The `zwavejs` binding supports auto-discovery of Z-Wave devices.
+When the bridge is added and connected, it will automatically discover and add Z-Wave nodes to the openHAB system.
+The following discovery features are available:
+
+- Automatic discovery of Z-Wave nodes when they are added to the network.
+- Automatic update of node information when the node is updated in the Z-Wave network.
+
+## Bridge Configuration
+
+The `zwavejs` binding requires configuration of the bridge to connect to the Z-Wave JS Webservice.
+The configuration options include:
+
+| Name                  | Type    | Description                                         | Default | Required | Advanced |
+|-----------------------|---------|-----------------------------------------------------|---------|----------|----------|
+| hostname              | text    | Hostname or IP address of the server                | N/A     | yes      | no       |
+| port                  | number  | Port number to access the service                   | 3000    | yes      | no       |
+| maxMessageSize        | number  | Maximum size of messages in bytes                   | 2097152 | no       | yes      |
+| configurationChannels | boolean | Expose the command class 'configuration' as channels | false   | no       | yes      |
+
+## Thing Configuration
+
+Each Z-Wave node can have its own configuration options.
+All configuration parameters are set by the binding during startup and are read-only.
+Only the advanced parameter `nodeId` can (optionally) be set in the Thing configuration in the openHAB UI or in the `.things` file.
+
+## Channels
+
+Z-Wave nodes can have multiple channels corresponding to their capabilities.
+The channels can be linked to items in openHAB to control and monitor the device.
+These channels are dynamically added to the Thing during node initialization; therefore, there is no list of possible channels in this documentation.
+
+## Full Example
+
+### `demo.things` Example
+
+```java
+Bridge zwavejs:gateway:myBridge "Z-Wave JS Bridge" [ hostname="localhost", port=3000 ] {
+    Thing node node1 "Z-Wave Node 1" [ id=1 ]
+    Thing node node2 "Z-Wave Node 2" [ id=2 ]
+}
+```
+
+### `demo.items` Example
+
+```java
+Switch LightSwitch "Light Switch" { channel="zwavejs:node:myBridge:node1:switch_binary" }
```

Comment:
Did you use _ in your channel ids ?
Or is it the example that is "wrong" ?
Maybe you should check the convention, if I correctly remember it is rather - that should be used for channels.

Reply:
Just a mistake in the readme, the convention is used as expected.